d3a_api_client/redis_aggregator.py

```diff
@@ -167,51 +168,28 @@ def batch_command(self, batch_command_dict, is_blocking=True):
 
     def _on_market_cycle(self, message):
         logging.info(f"A new market was created. Market information: {message}")
-
-        def executor_function():
-            try:
-                self.on_market_cycle(message)
-            except Exception as e:
-                root_logger.error(
-                    f"on_market_cycle raised exception: {str(e)}. \n Traceback: {str(traceback.format_exc())}")
-
-        self.executor.submit(executor_function)
+        function = lambda: self.on_market_cycle(message)
+        self.executor.submit(execute_function_util, function=function, function_name="on_market_cycle",
+                             )
```

Comment:
out of curiosity which editor are you using that suggests to place a closing parenthesis to a whole new line?

```suggestion
        self.executor.submit(execute_function_util, function=function, function_name="on_market_cycle")
```

Reply:
This is not always case, maybe it was already on a new line after I deleted the extra argument